examples/advanced/gnn/README.md

```diff
@@ -29,6 +29,11 @@ Install additional requirements:
 ```
 python3 -m pip install -r requirements.txt
 ```
+To further support PyTorch Geometric, we need to install in the following manner, please check the torch version and perform the installation accordingly:
+```
+pip install torch-scatter torch-sparse torch-cluster torch-spline-conv torch-geometric -f https://data.pyg.org/whl/torch-2.1.0+cpu.html
```

Comment:
suggest change this to the official documentation about installation:
https://pytorch-geometric.readthedocs.io/en/latest/install/installation.html

Reply:
good point, thanks! updated